index.bs

```diff
@@ -690,7 +699,7 @@ Note: In order to release hardware resources, the user agent can request underly
 A <dfn>sensor type</dfn> has an associated [=interface=]
 whose [=inherited interfaces=] contains {{Sensor}}.
 
-A [=sensor type=] has a [=ordered set|set=] of <dfn export>associated sensors</dfn>.
+A [=sensor type=] has a [=ordered set|set=] of associated [=device sensors|sensors=].
```

Comment:
should the set be non-empty?

Reply:
I guess it can be empty if the device does not have sensors of the given type.

Reply:
Actually it is not related to to this PR, I'll roll back this change.

Reply:
Sure, can be fixed in separate PR.